### PR TITLE
Buff Medical Due to Shitmed Changes Cherry-Pick

### PR DESCRIPTION
**Pull request by Tmanzxd**
# Description

+ Buffed applicable medication heal values.
+ Buffed applicable medication stack sizes.
+ Slightly buffed dylovene, burn, and brute chems.
+ Buffed applicable medication doAfter delay from 3s to 2s.

---

## Why / Balance

This change was made with the advent of the new Limb Damage system. This PR seeks to achieve making SS14 Medical more equivalent to SS13 Medical when it comes to healing damage values.

This was done as Bruise Packs and Ointment only heal 5 Brute or 5 Burn respectively when applied to a limb. When each limb has a 100 Crit threshold, this can make it extremely hard even with surgery to heal limbs in the absence of Chemicals.

Furthermore on higher pop, this will lead to people burning through applicable meds even faster than before Limb Damage was implemented. It would take 34 bruise packs to heal someone with 170 Blunt spread out across their limbs, with this change it would reduce that to 12 bruise packs.

Full list of numerical val changes:
 - Healing component doAfter Delay reduced. was 3s, now its 2s.
 - Applicable medical items now have a max stack size of 15 (was 10).
 - Ointment heals 15 of each burn type, 10 caustic (was 5 of each burn type, 1.5 caustic)
 - Mesh now heals 20 of each burn type (was 10 of each burn type).
 - Bruise Packs now heal 15 of each brute type (was 5 of each brute type)
 - Sutures now heal 20 of each brute type (was 10 of each brute type)
 - Bloodpacks now heal 2.5 Bloodloss & restore 10% bloodlevel (was .5 Bloodloss and 5%)
 - Gauzes now heal 15 Slash and 20 Pierce (was 5 Slash and 10 Pierce)
 - Dylovene now heals 1.5 Poison per .5u (was 1 Poison per .5u)
 - Bicaridine now heals 2.5 Brute per .5u (Was 2 Brute per .5u)
 - Dermaline now heals 2 Burn per .5u (was 1.5 Burn per .5u)
 - Epinephrine now heals 1 Burn & 1 Brute per .5u (was .5 for each per .5u)
 - Kelotane now heals .5 Burn per .5u (was .33 Burn per .5u)
 - Omnizine now heals 3 Burn, Toxin, Airloss, and Brute per .5u (was 2 per .5u)
- Lacerinol now heals 4 Slash per .5u (was 3 per .5u)
- Bruizine now heals 4 Blunt per .5u (was 3.5 per .5u) 

# TODO

- [x] Cherry-pick Medical changes merged in Goobstation

---

---
 
 ## Breaking changes

Should values be overtuned, they can be reduced in a future PR or before this PR is merged.

**Changelog**

:cl:
- tweak: Increased applicable medication heal values.
- tweak: Increased stack size of applicable medications.
- tweak: Slightly increased dylovene, burn, and brute chemicals heal values.
- tweak: Decreased Medical item application time from 3s to 2s

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml
Comment (sleepyyapril) [inline Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml]: Maybe put it to 50? Sutures are rare and hard to get outside of getting logi to buy you 30 of them. It's only fair, given the surgery's ability to remove most "very painful" damage.

File: Resources/Prototypes/Reagents/medicine.yml
Comment (sleepyyapril) [inline Resources/Prototypes/Reagents/medicine.yml]: Perhaps look into buffing diphenhydramine and stelli by 1.5-2x? Considering that replacing a liver (incredibly easy) now heals it instantly, you have to think about either nerfing that or changing to match.

**Review by sleepyyapril — APPROVED** [review]
I'd also highly consider changing cryo to be better, especially in terms of oppo. 
Also, test fails are valid. I like the changes, but do fix the fails.

**Comment by sleepyyapril** [comment]
Test fails are also valid.

File: Resources/Prototypes/Reagents/medicine.yml
Comment (gluesniffler) [reply Resources/Prototypes/Reagents/medicine.yml]: The current state of livers magically healing all poison damage is not intended to be the norm, just a funny and easy replacement for dialysis/blood pumps while i figure out how to deal with the procedures that would depend on a static structure (IV drips, dialysis, respirators, pumps, etc). 

But a buff would probably be in order anyway since they will be as efficient as having tend wounds for those specific damage types.

Comment (Tmanzxd) [reply Resources/Prototypes/Reagents/medicine.yml]: Diphenhydramine Heals 90u poison per 15u, Setllibinnin Heals 96 poison per 15u. I don't think a buff should be necessary for those at the time being.

File: Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml
Comment (Tmanzxd) [reply Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml]: That would increase the sale value which is the problem we're having with the test fails rn.

Comment (sleepyyapril) [reply Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml]: > That would increase the sale value which is the problem we're having with the test fails rn.

That isn't a massive concern though.

Comment (Tmanzxd) [reply Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml]: Fair, but a crate for 3 advanced kits costs $2k with the updates. I think that should be fine.

**Comment by Remuchi** [comment]
To me, this change seems a bit too radical. Buffing most medical items three times (somewhere even 6-7 times) makes them overtuned.
> It would take 34 bruise packs to heal someone with 170 Blunt 
It used to be the same before newmed. 

Medical items are sort of "last resort" of you really have no chemicals on the station OR the person is already dead and you have to get the damage below 200 so you can revive them and start applying chems. 
I'd advice that now it's even less need for medical items - since you can just threat wounds with surgery if the person is above 200 damage and that's it.

**Comment by Tmanzxd** [comment]
Treat wounds only heals 5 per DoAfter currently and I've had situations on Goobstation and Grimbly Station where I've had to treat people with 500+ damage spread out across their limbs. Even with the buffs and surgery this is a time consuming process.

**Comment by Remuchi** [comment]
Saving someone beyond 300 damage SHOULD BE time consuming process. Either that or toss them in cryo with whatever works on dead people and hope for the best.

**Comment by Tmanzxd** [comment]
Even with the buffs this would be a time consuming process yes. Cryo I didn't change at all because it's even more useful now. This is purely to make medications reflect their SS13 counterpart.

If the values are overtuned when compared to their SS13 counterpart they can be reduced before merge.

**Comment by gluesniffler** [comment]
> Saving someone beyond 300 damage SHOULD BE time consuming process. Either that or toss them in cryo with whatever works on dead people and hope for the best.

Personally I think trying to minmax balance at this point is a huge waste of time, especially when we've got pain and other things on the horizon that will absolutely throw any idea of "balancing" or "should" out of the window for 14.

**Comment by sleepyyapril** [comment]
> Saving someone beyond 300 damage SHOULD BE time consuming process. Either that or toss them in cryo with whatever works on dead people and hope for the best.



> Saving someone beyond 300 damage SHOULD BE time consuming process. Either that or toss them in cryo with whatever works on dead people and hope for the best.

Funnily enough, I'm fairly sure cryo (at least oppo) is broken. It returns you to rotting too quick for it to be not a bug.

**Comment by Tmanzxd** [comment]
If anything I'm looking at the values when compared to SS13 rn, the current values I have heal 5 more than their SS13 counterpart. But I have seen people walk into medbay with >400 burn damage on each of their limbs. So this is a call by the maintainers.

**Comment by Tmanzxd** [comment]
Talked to others about it, gonna reduce the values by 5 to reflect their SS13 counterpart more so its properly tuned.

**Comment by Tmanzxd** [comment]
There we go, that should fix the tuning.

**Review by VMSolidus — APPROVED** [review]

**Pull request merged**
